### PR TITLE
fix build

### PR DESCRIPTION
**Pull request by wgwz**
## Description
Closes: regen-network/regen-registry#1320
Based on: https://github.com/regen-network/regen-web/pull/1519

- Node build version to `lts/gallium` (16)
- increases from 4096 to 10216 `NODE_OPTIONS: "--max-old-space-size=10216"` to prevent hanging builds due to lack of memory

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood`, `hambach` and optionally `master` (see below)

#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -5,19 +5,22 @@ on:
     branches: [ master ]
   pull_request:
 
+env:
+  NODE_OPTIONS: "--max-old-space-size=10216"
```
Comment (wgwz) [inline .github/workflows/main.yml:9]: prevents hanging github action ci builds

File: netlify.toml [hunk 2]
```diff
@@ -55,7 +55,7 @@
   REACT_APP_LEDGER_CHAIN_ID = "regen-redwood-1"
   REACT_APP_LEDGER_CHAIN_NAME = "Regen - Redwood Testnet"
   REACT_APP_BLOCK_EXPLORER = "https://redwood.regen.aneka.io"
-  REACT_APP_SENTRY_ENVIRONMENT="redwood"
+  REACT_APP_SENTRY_ENVIRONMENT = "redwood"
 
 [build.environment]
-  NODE_OPTIONS = "--max_old_space_size=4096"
+  NODE_OPTIONS = "--max-old-space-size=10216"
```
Comment (wgwz) [inline netlify.toml:61]: prevents hanging netlify builds

File: netlify.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [context.production.environment]
-  REACT_APP_SANITY_TAG="default"
+  REACT_APP_SANITY_TAG = "default"
```
Comment (haveanicedavid) [inline netlify.toml:2]: does the whitespace here matter?

**Review by haveanicedavid — APPROVED** [review]

File: netlify.toml (hunk 3 above)
Comment (mhagel) [reply netlify.toml:2]: I don't think so, but this makes it consistent with the rest of the file

**Review by mhagel — APPROVED** [review]

**Comment by wgwz** [comment]
@Mergifyio backport redwood hambach master

**Pull request merged**